pkg/auth/iam/iam_test.go

```diff
@@ -818,369 +817,3 @@ func TestGetHost(t *testing.T) {
 		})
 	}
 }
-
-func TestValidateRefererHeaderWithSubdomainFilteringFlag_SimpleRedirectURI(t *testing.T) {
```

Comment:
Why removing test?

Reply:
those test cases were added after I added a code to read env var in the NewFilter.
now that we don't use it anymore, I'm deleting it.